semgrep/semgrep/commands/scan.py

```diff
@@ -251,8 +251,8 @@ def convert(
     "Use --config auto to automatically obtain rules tailored to this project; your project URL will be used to log in"
     " to the Semgrep registry."
     "\n\n"
-    "To run multiple rule files simultaneously, use -f before every YAML, URL, or Semgrep registry entry name."
-    " For example `semgrep -f your/java.net.url. -f path/secrets.`"
+    "To run multiple rule files simultaneously, use --config before every YAML, URL, or Semgrep registry entry name."
+    " For example `semgrep --config p/java --config p/secrets.`"
```

Comment:
```suggestion
    " For example `semgrep --config p/java --config p/secrets`"
```

Reply:
@minusworld this was an actual suggestion from @ievans, I think that myrules example may sound a bit nicer. :-) Let me know which one you choose, I edit the docs according to your decision here. 🙇

Reply:
All good for me 👍